src/structlog/_log_levels.py

```diff
@@ -60,10 +60,15 @@ def add_log_level(
     .. versionchanged:: 20.2.0
        Importable from `structlog.processors` (additionally to
        `structlog.stdlib`).
+    .. versionchanged:: 24.1.0
+       Added mapping from "exception" to "error"
     """
     if method_name == "warn":
         # The stdlib has an alias
         method_name = "warning"
+    if method_name == "exception":
```

Comment:
```suggestion
    elif method_name == "exception":
```

this saves us one if-check per warn ;)